src/commands/edge_app.rs

```diff
@@ -415,9 +415,12 @@ impl EdgeAppCommand {
         manifest: &EdgeAppManifest,
     ) -> Result<(), CommandError> {
         const SLEEP_TIME: u64 = 2;
-        const MAX_WAIT_TIME: u64 = 20; // 20 seconds
+        const MAX_WAIT_TIME: u64 = 1000; // 1000 seconds - it could take a while for assets to process
         let mut total_duration = 0;
 
+        let mut pb: Option<ProgressBar> = Option::None;
+        let mut assets_amount = 0;
```

Comment:
assets_number? assets_len? I am not even sure what this var immediately means? Processed amount?